tutorial/interpreter.po

```diff
@@ -83,13 +87,14 @@ msgid ""
 msgstr ""
 "Les fonctionnalités d'édition de l'interpréteur comprennent l'édition "
 "interactive, la substitution depuis l'historique et la complétion sur les "
-"systèmes qui gèrent *readline*. Un moyen rapide de tester comment est gérée "
-"l'édition de la ligne de commande, c'est de taper :kbd:`Control-P` à la "
-"première invite de commande que vous rencontrez. Si cela bipe, vous disposez "
-"de l'édition de la ligne de commande ; lisez l'appendice :ref:`tut-"
+"systèmes qui prennent en charge la bibliothèque `GNU Readline <https://tiswww.case."
+"edu/php/chet/readline/rltop.html>`_. Un moyen rapide de tester si "
+"l'édition de la ligne de commande gérée consiste à taper :kbd:`Control-P` dans "
+"la première invite de commande que vous rencontrez. Si cela bipe, vous "
+"disposez de l'édition de la ligne de commande ; lisez l'appendice :ref:`tut-"
```

Comment:
```suggestion
"disposez de l'édition de la ligne de commande ; lisez l'appendice :ref:`tut-"
```

/tmp/padpo_zivx6y2p/tutorial/interpreter.po:  76 [     NBSP     ] There should be a non-breakable space before ";": between ###dition de la ligne de commande ### and ###; lisez l'appendice « tut-inter###